app/gamelib/src/game_state/systems/actions.rs

```diff
@@ -38,7 +63,7 @@ mod tests {
         assert_eq!(state.is_running, true);
 
         // Pressing the home button should exit the game.
-        state.changes.controls.home_button_down = true;
+        state.changes.controls.wii_mote_control[0].home_button_down = true;
```

Comment:
Maybe we could create a numbered enum for the controllers? That way we can't create accidental overflow mistakes. Like `state.changes.controls.wii_mote_control[Controllers::One].home_button_down = true;`. Just an idea though, feel free to disagree.

Reply:
Hmm I see the point, but still not sure if I like it more